tests/e2e/lib.sh

```diff
@@ -780,25 +792,41 @@ remove_existing_stackrox_resources() {
     info "Tearing down StackRox resources for namespaces ${namespaces[*]}..."
 
     # Check API Server Capabilities.
-    if kubectl api-resources -o name | grep -q "^securitycontextconstraints\.security\.openshift\.io$"; then
+    local k8s_api_resources
+    k8s_api_resources=$(kubectl api-resources -o name)
+    if echo "${k8s_api_resources}" | grep -q "^securitycontextconstraints\.security\.openshift\.io$"; then
         resource_types="${resource_types},SecurityContextConstraints"
     fi
-    if kubectl api-resources -o name | grep -q "^podsecuritypolicies\.policy$"; then
+    if echo "${k8s_api_resources}" | grep -q "^podsecuritypolicies\.policy$"; then
         psps_supported=true
         global_resource_types="${global_resource_types},psp"
     fi
+    if echo "${k8s_api_resources}" | grep -q "^centrals.platform.stackrox.io$"; then
+        centrals_supported=true
+    fi
+    if echo "${k8s_api_resources}" | grep -q "^securedclusters.platform.stackrox.io$"; then
```

Comment:
Escape the dots here for correctness and consistency too?

Reply:
right